<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDASAccountPasswordDialog</class>
 <widget class="QDialog" name="CMDASAccountPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>帐户最小长度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minAccountLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>密码最小长度</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>410</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码有效期 [0=密码永不过期]</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_PswValidPeriod">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comb_PswValidPeriodUnit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>分钟</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>小时</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>月</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>连续N次输入错误密码后，自动锁定[0=不自动锁定]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>N=</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_invalidPswCount">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>次</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>经过以下时间后自动解锁[0=必须手动解锁]</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_AutoUnlockPeriod">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comb_AutoUnlockUnit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>分钟</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>小时</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>月</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="cb_deleteAccount">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>允许删除账户</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_CantUseOldPsw">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>不可使用旧密码</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>280</y>
     <width>451</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QCheckBox" name="cb_Alphabet">
      <property name="text">
       <string>包含字母</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_Number">
      <property name="text">
       <string>包含数字</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_UppercaseLowercase">
      <property name="text">
       <string>包含大小写</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_Extra_Alphabet">
      <property name="text">
       <string>包含特殊字符</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>410</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comb_LockScreenPeriodUnit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>分钟</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>小时</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>月</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>锁屏时间[0=不启用锁屏功能]</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_LockScreenPeriod">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_modifyPasswordFirstLogin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>首次登陆必须修改密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_PswCannotContainAccout">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>密码不能包含账户名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
